pkg/kv/kvserver/metrics.go

```diff
@@ -1221,6 +1221,37 @@ histogram.
 		Measurement: "Processing Time",
 		Unit:        metric.Unit_NANOSECONDS,
 	}
+	metaRaftCommandsProposed = metric.Metadata{
+		Name: "raft.commands.proposed",
+		Help: `Count of Raft commands proposed.
+
+This measurement is taken on the leaseholders, and includes both the original
+proposals and re-proposals. This approximates the number of commands submitted
+through Raft.`,
+		Measurement: "Commands",
+		Unit:        metric.Unit_COUNT,
+	}
+	metaRaftCommandsReproposed = metric.Metadata{
+		Name: "raft.commands.reproposed.same",
+		Help: `Count of Raft commands re-proposed without modification.
```

Comment:
I'm not sure metric help is the right place to explain that proposals are done on leaseholder and why we make reproposals. Something like:

Number of Raft commands re-proposed without modification by leaseholder. These re-proposals happen when command is not committed/applied within timeout and is assumed lost and needs to be resent.